pkg/cmd/run.go

```diff
@@ -116,6 +117,10 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 
 		secrets := getSecrets(localConfig, enableFallback, fallbackPath, fallbackReadonly, fallbackOnly, exitOnWriteFailure, passphrase)
 
+		if preserveEnv {
+			utils.LogWarning("Ignoring any secrets already defined in the environment due to --preserve-env flag")
+		}
+
 		env := os.Environ()
 		excludedKeys := []string{"PATH", "PS1", "HOME"}
```

Comment:
Unrelated but why do we exclude these secrets?

Reply:
IIRC these were copied over from the node cli